docs/_common/install-otterize-from-cloud-with-shadow-mode-and-kafka-watcher-and-cert-manager.md

```diff
@@ -0,0 +1,12 @@
+Head over to the [Clusters page](https://app.otterize.com/clusters) and create a cluster.
+Follow the connection guide that opens to connect your cluster, and make the following changes:
+![Cluster connection guide](/img/configure-cluster/connect-cluster-kafka-mtls-with-otterize-cloud-cert-manager.png)
```

Comment:
The instructions should come before the screenshot, then the screenshot should say "the result should look like this" as a caption